README.md

```diff
@@ -28,19 +28,18 @@ yarn build          # Run dev
 yarn serve          # Serve the prod build
 ```
 
-## Configure
+## Contributing
 
-### Algolia
+We welcome contributions! Follow this [link](https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-docs/src/main/asciidoc/appendix-contributing.adoc) for more information on how to contribute.
 
-```bash
-export ALGOLIA_ADMIN_KEY=<KEY>
-```
+## Q&A and issue tracking
+
+If you have any feedback, or feature requests, don't hesitate to [add an issue](https://github.com/spring-io/dataflow.spring.io/issues).
```

Comment:
Maybe, instead of "feature requests", we can rephrase it to include the guides or documentation as the context.